commit 05efb061f1945425d214ff6b6050883e6d8633bf
Author: Koen Kooi <koen@dominion.thruhere.net>
Date:   Wed Jan 26 13:21:12 2011 +0100

    blacklist tv out

    The tv-out gets added last and the driver tries to reconfigure dvi to TV resolutions, which fails. so blacklist it

	Upstream-Status: Pending

    Signed-off-by: Koen Kooi <koen@dominion.thruhere.net>

diff --git a/src/omapfb-output-dss.c b/src/omapfb-output-dss.c
index 6cc52de..83cb711 100644
--- a/src/omapfb-output-dss.c
+++ b/src/omapfb-output-dss.c
@@ -205,6 +205,10 @@ OMAPFBDSSOutputDetect (xf86OutputPtr output)
 	if(ofb->timings[idx][0] == '\0')
 		return XF86OutputStatusDisconnected;
 
+	// Hack to disable the tv out
+	if (strncmp(output->name, "tv", 2) == 0)
+		return XF86OutputStatusDisconnected;
+
 	return XF86OutputStatusConnected;
 }
 
